decentraland/marketplace
# feat: new activity page

This PR uses the features from the new `decentraland-dapps` transaction module to improve the information we display in the activity page:

<img width="727" alt="screen shot 2018-09-28 at 5 30 51 pm" src="https://user-images.githubusercontent.com/2781777/46232621-13ac8300-c346-11e8-982d-e613c035423b.png">

Transactions now display `Loading`, `Queued`, `Replaced` and `Dropped` statuses, with tooltip information and also the amount of confirmations with a circular progress bar (which has to reach 20 to be considered accepted).

# How to test 

The easiest way to test this is to edit a parcel name several times.

To force `Queued` transactions send a transaction with 0.1 Gwei and then a couple more with regular gas price.

To force `Failed` transactions click a bunch of times on the `Authorize the Marketplace to operate LAND on your behalf` checkbox in the settings page (without confirming the metamask popups) then confirm all the popups all at once (the first tx will work, the next ones will fail).

To force `Dropped` and `Replaced` transactions send a transaction with low gas price (0.1 Gwei) and after a minute metamask will let you replace it. You should see the transaction go from `Pending` to `Dropped` until the transaction is mined, then it should change to `Replaced` (after the 20 confirmations).

To force the `Loading` status send a transaction with a reaaaally low gas price (0.001 Gwei) it should stay loading for a while.
